javascript/froogaloop.js

```diff
@@ -208,7 +208,7 @@ var Froogaloop = (function(){
      * Retrieves stored callbacks.
      */
     function getCallback(eventName, target_id) {
-        if (target_id) {
+        if (target_id && eventCallbacks[target_id] != undefined) {
```

Comment:
I would change this to: `if (target_id && typeof eventCallbacks[target_id] !== 'undefined') {`